data/data/agent/files/etc/udev/rules.d/80-agent-config-image.rules

```diff
@@ -0,0 +1 @@
+ACTION=="add", SUBSYSTEM=="block", ENV{ID_FS_LABEL}=="agent_configimage", ENV{SYSTEMD_WANTS}+="load-config-iso@$devnode.service"
```

Comment:
> Just wondering, why is the load-config-iso defined as an instanced service unit? I.e. what's the reasoning for naming it with a '@' postfix?

This is necessary so that udev can start the host with the devname of the config-image. For example, the service below is created when the config-image is mounted on /dev/sr1
```
$ sudo systemctl status load-config-iso@-dev-sr1.service
● load-config-iso@-dev-sr1.service - If a config ISO is mounted, copy its files to the system
     Loaded: loaded (/etc/systemd/system/load-config-iso@.service; disabled; preset: disabled)
     Active: activating (start) since Tue 2023-06-06 13:00:21 UTC; 4min 3s ago
   Main PID: 1581 (load-config-iso)
      Tasks: 2 (limit: 204700)
     Memory: 1.0M
        CPU: 956ms
     CGroup: /system.slice/system-load\x2dconfig\x2diso.slice/load-config-iso@-dev-sr1.service
             ├─1581 /bin/bash /usr/local/bin/load-config-iso.sh /dev/sr1
             └─8764 sleep 5
```